jboss-kie-wildfly-common/added/launch/jboss-kie-wildfly-security.sh

```diff
@@ -180,41 +97,12 @@ function add_kie_server_controller_user() {
 print_user_information() {
     if [ "${AUTH_LDAP_URL}x" != "x" ] || [ "${SSO_URL}x" != "x" ]; then
         log_info "External authentication/authorization enabled, skipping the embedded users creation."
-        if [ "${1}" == "kieadmin" ] || [ "${1}" == "central" ] || [ "${1}" == "kieserver" ]; then
-            if [ "${KIE_ADMIN_USER}x" != "x" ]; then
-                log_info "KIE_ADMIN_USER is set to ${KIE_ADMIN_USER}, make sure to configure this user with the provided password on the external auth provider with the roles $(get_kie_admin_roles)"
-            else
-                log_info "Make sure to configure a ADMIN user to access the Business Central with the roles $(get_kie_admin_roles)"
-            fi
-        fi
-
-        if [ "${1}" == "central" ] || [ "${1}" == "kieserver" ]; then
-            if [ "${KIE_MAVEN_USER}x" != "x"  ]; then
-                log_info "KIE_MAVEN_USER is set to ${KIE_MAVEN_USER}, make sure to configure this user with the provided password on the external auth provider."
-            else
-                log_info "Make sure to configure the KIE_MAVEN_USER user to interact with Business Central embedded maven server"
-            fi
-
-        fi
-
-        if [ "${1}" == "central" ] || [ "${1}" == "kieserver" ] || [ "${1}" == "controller" ]; then
-            if [ "${KIE_SERVER_CONTROLLER_USER}x" != "x" ]; then
-                log_info "KIE_SERVER_CONTROLLER_USER is set to ${KIE_SERVER_CONTROLLER_USER}, make sure to configure this user with the provided password on the external auth provider with the roles $(get_kie_server_controller_roles)"
-            else
-                log_info "Make sure to configure the KIE_SERVER_CONTROLLER_USER user to interact with KIE Server rest api with the roles $(get_kie_server_controller_roles)"
-            fi
-
-        fi
-
-        if [ "${1}" == "kieserver" ] || [ "${1}" == "controller" ]; then
-            if [ "${KIE_SERVER_USER}x" != "x" ]; then
-                log_info "KIE_SERVER_USER is set to ${KIE_SERVER_USER}, make sure to configure this user with the provided password on the external auth provider with the roles $(get_kie_server_roles)"
-            else
-                log_info "Make sure to configure the KIE_SERVER_USER user to interact with KIE Server rest api with the roles $(get_kie_server_roles)"
-            fi
+        if [ "${KIE_ADMIN_USER}x" != "x" ]; then
```

Comment:
@ruromero Is print_user_information() still needed? If I looked correctly is it no more used in scripts.
If we would like to have it include as an info log when user is created in any BA component, we should add print_user_information() back as it was removed in PR #348 
Also can you please check tests if they are passing? Because user information print was removed from workbench as well - see [in PR here](https://github.com/jboss-container-images/jboss-kie-modules/pull/348/files#diff-15def47866cc98f6716dd20cf552cabbL73)

Reply:
It is indeed dead code, let me know what do you think about this PR https://github.com/jboss-container-images/jboss-kie-modules/pull/354